hasweb/pages/site/hgtos.md

```diff
@@ -1,36 +1,38 @@
 
 The term “HasGeek Media LLP” or “us” or “we” refers to the owner of the website whose registered office is No 141/142, 2nd Cross, Pai Layout, Hulimavu Gate, Bannerghatta Road, Bangalore - 560 076, Karnataka, India. Our registration number is AAA-3107 and we are incorporated under the Limited Liability Partnership Act 2008 of India. 
 
-We take great care to produce outstanding code, render services and/or products. But there is a great chance the code will have bugs, products will not work exactly as you expect and there is the small chance you may be unhappy with our services. In any event, that cannot be grounds to sue us in a court of law. In legal speak, this service is provided on an AS IS, WHERE IS basis. No warranties, no liability. Except of course, if the law otherwise provides.
+We take great care to produce outstanding code, render services and/or products. But there is a great chance the code will have bugs, products will not work exactly as you expect and there is the small chance you may be unhappy with our services. Neither we nor any third parties provide any warranty or guarantee as to the accuracy, timeliness, performance, completeness or suitability of the information and materials found or offered on this website for any particular purpose. This service is provided on an AS IS, WHERE IS basis. No warranties, no liability. Except of course, if the law otherwise provides. You acknowledge that such information and materials may contain inaccuracies or errors and we expressly exclude liability for any such inaccuracies or errors to the fullest extent permitted by law.
 
 HASGEEK SHALL NOT BE LIABLE FOR ANY SPECIAL, INCIDENTAL, INDIRECT OR CONSEQUENTIAL DAMAGES OF ANY KIND IN CONNECTION WITH THIS AGREEMENT, EVEN IF HASGEEK HAS BEEN INFORMED IN ADVANCE OF THE POSSIBILITY OF SUCH DAMAGES.
 
-We charge for certain services and provide other services without charge. But whether we do charge or not is a commercial decision and we believe it is important that we reserve the right to change pricing and relating policies if we feel the need to do so. But we will alert you when we are about to effect such changes.
+We charge for certain services and provide other services without charge, this is a commercial decision and we believe it is important that we reserve the right to change pricing and relating policies if we feel the need to do so. But we will alert you when we are about to effect such changes. Refer to our product specific ToS for refund and cancellation policies.
 
-## Acceptable Use
+Your use of this website is subject to the following guidelines:
 
-Your use under these terms must not violate applicable laws, must comply with our content guidelines (which we may vary from time to time) and is subject to deletion or suspension if we receive complaints from third parties. Here are our [Acceptable Use Guidelines](terms)
+## Content
 
-## Communications
```

Comment:
This was about HasGeek using email addresses of users on file to send out mailers. It has been mentioned in privacy.md